Clinical-Genomics/scout
# Add browser analytics

This PR adds a call to Netlify passing the user agent (containing browser name and operative system) and screen size when clicking on login. This is a temporary solution to understand which browsers are being used.


**How to test**:
Go to staging, deploy this branch. When logging in you will see an http call to Netlify passing the data


**Expected outcome**:
The functionality should be working

**Review:**
